marketplaceItems/audioFocus/appResources/appData/audioFocus_app.js

```diff
@@ -0,0 +1,459 @@
+                position = Vec3.sum(MyAvatar.position, Vec3.multiplyQbyV(MyAvatar.orientation, MyAvatar.getAbsoluteJointTranslationInObjectFrame(controllerJointIndex)));
+
+                // add to the real position so the grab-point is out in front of the hand, a bit
+                if (doOffset) {
+                    var offset = getGrabPointSphereOffset(handController);
+                    position = Vec3.sum(position, Vec3.multiplyQbyV(orientation, offset));
+                }
+
+            } else if (!HMD.isHandControllerAvailable()) {
+                // NOTE: keep this offset in sync with scripts/system/controllers/handControllerPointer.js:493
+                var VERTICAL_HEAD_LASER_OFFSET = 0.1 * MyAvatar.sensorToWorldScale;
+                position = Vec3.sum(Camera.position, Vec3.multiplyQbyV(Camera.orientation, { x: 0, y: VERTICAL_HEAD_LASER_OFFSET, z: 0 }));
+                orientation = Quat.multiply(Camera.orientation, Quat.angleAxis(-90, { x: 1, y: 0, z: 0 }));
+                valid = true;
+            }
+        }
+
+        return {
+            position: position,
+            translation: position,
+            orientation: orientation,
+            rotation: orientation,
+            valid: valid
+        };
+    }
+
+    // #endregion
+    // *************************************
+    // END UTILITY FUNCTIONS
+    // *************************************
+
+
+    // *************************************
+    // START INIT
+    // *************************************
+    // #region Init
+
+    var isEnabled = true;
+    var soloAvatars = {};
+
+    // #endregion
+    // *************************************
+    // STOP INIT
+    // *************************************
+
+
+    // *************************************
+    // START MAPPING FUNCTIONS
+    // *************************************
+    // #region Mapping
+
+    var lastMouseX;
+    var lastMouseY;
+    // Record the last mousePressEvent
+    function mousePressEvent(event) {
+        lastMouseX = event.x;
+        lastMouseY = event.y;
+    }
+
+
+    // Enables all the pointers created and the mapping
+    function enable() {
+        Controller.enableMapping(MAPPING_NAME);
+    }
+
+
+    // Disables all the pointers created and the mapping 
+    function disable() {
+        Controller.disableMapping(MAPPING_NAME);
+    }
+
+
+    // Returns the right UUID based on hand triggered
+    function getUUIDFromLaser(hand){
+        hand = hand === Controller.Standard.LeftHand 
+            ? Controller.Standard.LeftHand
+            : Controller.Standard.RightHand;
+        
+        var pose = getControllerWorldLocation(hand);
+        var start = pose.position;
+        var direction = Vec3.multiplyQbyV(pose.orientation, {x:0, y:1, z: 0});
+
+        var avatarIntersection = AvatarList.findRayIntersection({origin:start, direction:direction});
```

Comment:
```suggestion
        var avatarIntersection = AvatarList.findRayIntersection({origin: start, direction: direction});
```

Reply:
also: is there any reason you're not passing `MyAvatar.sessionUUID` as an argument to this to ignore a ray intersection with your avatar?